tox.ini

```diff
@@ -7,6 +7,7 @@ changedir = tmp
 setenv =
     MPLBACKEND = agg
     COLUMNS = 180
+    HOME = {homedir}
```

Comment:
I need to work out a better solution than this.

Reply:
With Astropy's `_find_home()` relying on the `HOME` environment variable (at least currently), I don't know what better solution there could be.  Why do you want to avoid creating this environment variable?

Reply:
I think the best thing to do is to use the [`set_temp_config`](https://github.com/astropy/astropy/blob/master/astropy/config/paths.py#L195) helper in astropy config so that we don't actually write to the users home directory who is running the tests.

The tests should be stateless themselves without having to rely on the clean VMs of the CI service.

Reply:
Okay, I see what you're saying, but doesn't SunPy currently assume that its config is under the user's home directory, so we're already broken?  It seems that you should create a new issue for this.

Reply:
We create a temp sunpy config during test runs.

Reply:
https://github.com/sunpy/sunpy/blob/master/sunpy/conftest.py#L40

Reply:
We do?  [This line](https://github.com/sunpy/sunpy/blob/master/sunpy/util/tests/test_config.py#L20) was causing a test failure when @nabobalis accidentally set `HOME` to an incorrect directory.

Reply:
I guess I don't understand the temp SunPy config if the directories aren't changed

Reply:
:confused: Yeah I don't know either.

Reply:
I think also that failure is due to just being a bad test. Normally on the CI where the clone repo is, is the same place as the user. But on windows for azure, the home user and the config location is are different.

Notice that everything is occurring under `D:\a\1\s\.tox\py36-offline`  where the USER directory is.
But for some reason the CONFIG_DIR is `C:\\Users\\VssAdministrator\\AppData\\Local\\sunpy\\sunpy` 

I suspect this could be a bug in how we set the dir for windows.

Reply:
In that test failure, `USER` is set to `D:\a\1\s\.tox\py36-offline` because you had set `HOME` to `{toxinidir}` instead of `{homedir}`, so it was a one-time error in the configuration

Reply:
OH it was the test from that. I thought it was the test results  when it was set correctly. Ignore me then.